decidim-admin/spec/forms/newsletter_form_spec.rb

```diff
@@ -36,12 +36,6 @@ module Admin
 
         it { is_expected.to be_invalid }
       end
-
-      context "when there's no body" do
```

Comment:
shouldn't we check if there's a template?

Reply:
There's no template yet at this point! The template is set at the command.